wAbout/Logging-In-Enterprise.md

```diff
@@ -0,0 +1,112 @@
+<br>
+
+**Step 3:** Enter your enterprise login code and click **Look Up Code**.
+
+![](/images/Login-Enterprise/EnterpriseLoginCode.png)
+<br>
+
+**Step 4:** Click your Identity Provider in the list.
+
+![](/images/Login-Enterprise/SelectProvider.png)
+<br>
+
+**Step 5:** Enter your federated Credentials and click **Continue**. 
+
+![](/images/Login-Enterprise/FederatedLogin.png)
+<br>
+
+You can verify you are logged in by the Login Manager Window.
+
+![](/images/Login-Enterprise/EnterpriseLoggedIn.png)
+<br>
+
+<blockquote class=highlight_note>
+<b>Note:</b> Some login pages may have a "Remember Me" or a "Save Login" checkbox to save time the next time the user comes to the login page. These features are not guaranteed to work while logging in using the Interject federated login approach. The browser enabling these federated login pages is Webview2 and not the user's standard browser and is not accessible from within the Excel Addin.
+</blockquote>
+
+### New Login
+
+**Step 1:** To add a new login, first click the **Login** button on the Interject Ribbon.
+
+![](/images/LogginIn/interject-ribbon-advanced-menu-login-revised.png)
+<br>
+
+**Step 2:** Click the **New Login** button and proceed to enter your credentials the same as in [initial enterprise login](#initial-enterprise-login).
+
+![](/images/Login-Enterprise/NewLogin.png)
+<br>
+
+### Logging Out
+
+Logging out means Interject will delete the login tokens and not automatically log you in the next time you return.
+
+To log out, you can click the ellipses on the account you wish to log out and click Log Out.
+
+![](/images/Login-Enterprise/Logout.png)
+<br>
+
+You may alternatively click the Logout button on the Interject ribbon.
+
+![](/images/LogginIn/RibbonLogout.png)
+<br>
+
+![](/images/LogginIn/LogoutPrompt.png)
+<br>
+
+### Log Back In
+
+To log back in after you have been logged out, simply click the ellipses on the account you wish to log in and click **Log In**.
+
+![](/images/Login-Enterprise/LogBackIn.png)
+<br>
+
+### Switch Accounts
+
+To switch between accounts, simply click the link of the account you wish to switch to.
+
+![](/images/Login-Enterprise/SwitchAccounts.png)
+<br>
+
+### Forgetting Login Credentials
+
+Forgetting credentials means the login token is deleted from the cache and the profile removed from the Login Manger. 
+
+To log out and forget your credentials, click the ellipses on the account you wish to log out and forget and click **Log Out & Forget**.
+
+![](/images/Login-Enterprise/LogoutForget.png)
+<br>
+
+### Clearing Login Cookies
+
+The Login cookies store the federated credentials so that the Enterprise user can log back in without having to reenter their federated credentials. To clear these cookies, select **Diagnostics** from the Interject ribbon, click on **Clear External Login Cookies** and then select **Execute Selected Action**. On the next login, you will have to reenter your federated credentials.
```

Comment:
enterprise here too. i don't see any need to cap it. but also it's not a hill i'll die on